rmf_task/include/rmf_task/requests/ChargeBattery.hpp

```diff
@@ -83,6 +83,29 @@ class ChargeBattery
     rmf_traffic::Time earliest_start_time,
     ConstPriorityPtr priority = nullptr,
     bool automatic = true);
+
+  /// Generate a chargebattery request.
+  ///
+  /// \param[in] earliest_start_time
+  ///   The desired start time for this request.
+  ///
+  /// \param[in] requester
+  ///   The entity that started this request.
```

Comment:
Nit: Consider replacing with `issued`
```suggestion
  ///   The entity that issued this request.
```

Reply:
https://github.com/open-rmf/rmf_task/pull/81/commits/bbb3689c63cbc89d5f17b299f5ba87071dc75164